examples/06_manipulating_images/plot_threshold_image.py

```diff
@@ -0,0 +1,175 @@
+# %%
+"""
+Image thresholding
+==================
+
+The goal of this example is to illustrate the use of the function
+:func:`~nilearn.image.threshold_img` together with ``threshold`` and
+``two_sided`` parameters to view certain values in image data.
+
+The ``threshold`` parameter can take both positive and negative values.
+``two_sided`` parameter is complementary to ``threshold`` effecting its
+behavior.
+
+"""
+
+# %%
+# Image without threshold
+# -----------------------
+#
+# We will first load the dataset and display the image without manipulation.
+
+from nilearn import datasets, plotting
+
+cmap = "RdBu_r"
+cut_coords = [-26, -33, 59]
+
+image = datasets.load_sample_motor_activation_image()
+
+plotting.plot_stat_map(
+    image,
+    colorbar=True,
+    cmap=cmap,
+    title="image without threshold",
+    cut_coords=cut_coords,
+)
+
+# %%
+# Image thresholded at 2 when two_sided=True
+# ------------------------------------------
+#
+# Now we will use ``threshold=2`` together with ``two_sided=True`` to
+# threshold the image. When ``two_sided=True``, we can only use positive
+# values for ``threshold``.
+#
+# This will set all image values between -2 and 2 to 0.
+
+import matplotlib.pyplot as plt
+
+from nilearn.image import threshold_img
+
+thresholded_img = threshold_img(
+    image,
+    threshold=2,
+    cluster_threshold=0,
+    two_sided=True,
+    copy=True,
+)
+
+
+fig, axes = plt.subplots(
+    1,
+    2,
```

Comment:
In general for this figure and  the following one let's switch to 2, 1 so the subplots stack on top of each other, to better take advantage of the screen width in the doc render.